File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCSAOCS0270Q00GridBAS0270RowInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    public class OCSAOCS0270Q00GridBAS0270RowInfo
    {
        private String _doctor;
        private String _doctorName;
        private String _contKey;
        private String _departmentName;

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String ContKey
        {
            get { return this._contKey; }
            set { this._contKey = value; }
        }

        public String DepartmentName
        {
            get { return this._departmentName; }
            set { this._departmentName = value; }
        }

        public OCSAOCS0270Q00GridBAS0270RowInfo() { }

        public OCSAOCS0270Q00GridBAS0270RowInfo(String doctor, String doctorName, String contKey, String departmentName)
        {
            this._doctor = doctor;
            this._doctorName = doctorName;
            this._contKey = contKey;
            this._departmentName = departmentName;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCSAOCS0270Q00GridBAS0270RowInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCSAOCS0270Q00GridBAS0270RowInfo {
	private String doctor;
    private String doctorName;
    private String contKey;
    private String departmentName ;
	public OCSAOCS0270Q00GridBAS0270RowInfo(String doctor, String doctorName, String contKey, String departmentName) {
		super();
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.contKey = contKey;
		this.departmentName = departmentName;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getContKey() {
		return contKey;
	}
	public void setContKey(String contKey) {
		this.contKey = contKey;
	}
	public String getDepartmentName() {
		return departmentName;
	}
	public void setDepartmentName(String departmentName) {
		this.departmentName = departmentName;
	}
	
}
